### ISSUE TITLE
Performance difference using previous and new releases..

### ISSUE THREAD

**Issue by sbaek15** [vlfeat/matconvnet#941]
I'm using simpleNN.
It seems the performance and convergence speed are different for me, when using the new release (version 1.0-beta24) and previous version for finetuning a network (AlexNet) with Oxford Pet dataset. (using same learning rate, batch size, architecture, etc)

Convergence speed reduced compared to previous version, even with batch normalization. Also, the performance is bad compared to previous version. I will double-check this again but, is there any new component that can cause this problem?
